tools/build/ubuntu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh 
+apt-get install --no-install-recommends --yes build-essential cmake doxygen \
+    git libboost-all-dev libcoin80c libcoin-dev libeigen3-dev libgtkglext1-dev \
```

Comment:
`libboost-all-dev` will pull in a lot of unnecessary dependencies, the required list is:
`libboost-date-time-dev libboost-dev libboost-filesystem-dev libboost-graph-dev libboost-iostreams-dev libboost-program-options-dev libboost-python-dev libboost-regex-dev libboost-serialization-dev libboost-thread-dev`